lib/ecto/association.ex

```diff
@@ -12,16 +12,16 @@ defmodule Ecto.Association.NotLoaded do
   """
 
   @type t :: %__MODULE__{
-    __field__: atom(),
```

Comment:
formatting kicked in. I'll need to explore what I need to do to get this to not be the case.

Reply:
Yes, please disable the formatter in your editor when working on Ecto. :)